playground/frontend/playground_components/LICENSE

```diff
@@ -200,208 +200,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
```

Comment:
The LICENSE file in this package was copied from the root LICENSE file of the entire Beam. After the Apache license it includes licenses for other files found elsewhere. They do not affect the `playground_components` package which if file with Apache license alone.

By the way, since it is not a separate publishable package, it does not need a license file at all, so I deleted it. Thanks.